doc/implementation-guide/client-direct.md

```diff
@@ -10,19 +10,24 @@ There are three main ways how to integrate *FAVI Partner Events Tracking* into y
 2. [client-side (frontend) integration using Google Tag Manager (called GTM)](client-gtm.md),
 3. [server-to-server integration using API (called S2S)](server-to-server.md).
 
-The S2S solution is most feature rich, reliable and secure, but requires direct integration into your backend code, which is not feasible for everyone. So pick the integration which most suits your situation.
+One of the client side solutions is needed to measure campaign performance - such as attribution.
+
+The S2S solution is most reliable and secure and allows you to  but requires direct integration into your backend code, which is not feasible for everyone.
+
+You can pick any solution, or even combination of them - for example it is best to use a client side solution to create an order, but you can send more (or update) information about it later using the S2S solution.
```

Comment:
Nenapíšeme, že best je použít všechny eventy na clientu a současně také S2S `pro more (or update) information about it later using the S2S solution` a protože na clientu to může být blokováno někdy?